config/enum.php

```diff
@@ -26,6 +26,21 @@
         'admin'     => 'Admin',
     ],
 
+    /*
+    |--------------------------------------------------------------------------
+    | Views
+    |--------------------------------------------------------------------------
+    |
+    | to be used in views
+    |
+    */
+
+    'navigation_states' => [
```

Comment:
NOOOOOOOOOOO. I thought we talked about why this is a terrible idea!!

Just add it to the Model. Stop doing this. PLEASE.

Reply:
This is one item that I would prefer as it is for now. Please.